public/index.html

```diff
@@ -58,6 +58,15 @@
     <div class="container">
         <div class="row">
             <div class="col-lg-8" id="plot-column">
+                <div id="neighbourplot-options" class="row hidden">
+                    <label for="dimensionality-reduction-select">
+                        Dimensionality Reduction:
+                    </label>
+                    <select id="dimensionality-reduction-select" class="form-control">
+                        <option value="tnse">TSNE</option>
```

Comment:
LOL tnse -> tsne

Where is the mongo query for this? I'm updating `microscopium` to create the tSNE features, so I want to make sure the schema matches.